kv/txn_test.go

```diff
@@ -201,9 +199,7 @@ func verifyUncertainty(concurrency int, maxOffset time.Duration, t *testing.T) {
 			// hasn't changed (i=0). The logical component will change
 			// internally in a way we can't track, but we want to be just
 			// ahead.if
```

Comment:
We want to avoid having the internal clock tick ahead of our timestamp here, so we blindly add 999 ticks hoping that's enough. Should probably take another look at it; it doesn't seem very sane.